libc/src/__support/wchar/character_converter.cpp

```diff
@@ -22,13 +25,65 @@ bool CharacterConverter::isComplete() {
   return state->bytes_processed == state->total_bytes;
```

Comment:
An empty state is considered complete?

If this is desired, it should be documented as a comment above the declaration in the .h file.